src/containers/Dashboard/Tutorials/Tutorial.js

```diff
@@ -1,14 +1,54 @@
 import React, { Component, Fragment } from "react";
-import {
-  INTERVIEW_PREP_KIT,
-  STATISTICS,
-  JAVASCRIPT,
-  CODE
-} from "../../../constants";
+// import {
+//   INTERVIEW_PREP_KIT
+// } from "../../../constants";
 import "./Tutorial.scss";
 import TutorialCard from "./TutorialCard";
+import { addNotification } from "../../../utilities";
+import API from "../../../api/index";
 
 class Card extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      tutorials: [],
+      tutorial : []
+    };
+  }
+
+  componentWillMount() {
+    API.getTutorials(result => {
+      if (result.status === "200") {
+        this.setState({
+          tutorials: result.data
+        });
+        if (this.state.tutorials.length < 6) {
+          this.state.tutorials.map((tutorial, index) => {
+            this.setState({
+              tutorial: this.state.tutorial.concat(tutorial.name)
+            });
+          })
+        } else {
+          for(let i =0 ; i<6; i++) {
```

Comment:
@HarisShahidHameed now need of of loop here. You have made it over complex.
Please coordinate with @Humnaa on this. Thanks!

Reply:
Removed the loops.